yury-ortuno/lesson-1/homework-2-Car/src/Car.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Created by Yury Ortuno on 11/13/2018.
+ */
+public class Car {
+    private String mark;
```

Comment:
This field is probably not relevant for our problem

Reply:
I use the field to show results